Steam Web Integration.user.js

```diff
@@ -437,6 +437,7 @@ function integrate(userdata, decommissioned, cards, bundles, limited, dlc, lastC
         `[href*="steamdb.info/app/"]`,
         `[href*="store.steampowered.com/agecheck/app/"]`,
         `[href*="store.steampowered.com/app/"]`,
+        `[href*="s.team/a/"]`,

```

Comment:
Make sure to inspect the `doApp()` function, since it expects the URL to have `/app(s)` included, if I'm not mistaken.

Reply:
I make a fix and it worked.
![image](https://user-images.githubusercontent.com/5698157/111018809-1c9a4d80-83f6-11eb-9568-b79453984b5d.png)